brave/brave-core
# feat: add portfolio graph visibility toggle button

Resolves https://github.com/brave/brave-browser/issues/26080

## Submitter Checklist:


## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on

## Test Plan:

Portfolio Page
1. Unlock wallet
2. navigate to portfolio screen
3. click the "eye" icon to on the right
4. The graph and timeline controls should toggle to appear/hide

Asset Details Page
1. Unlock wallet
2. navigate to portfolio screen
3. Click and asset (ie. Ether) to navigate to the asset details page
5. click the "eye" icon on the right of the accounts table header
6. The asset balances should toggle to hide/appear

https://user-images.githubusercontent.com/30185185/196491901-6554329b-376f-439a-b5b3-3ce44c14bac8.mov
